### PR TITLE
Remove overview page

### PR DESCRIPTION
**Pull request by gethyn1**
Remove “Overview” page and replace with “Guidance” page as first page of the application process:

- Remove overview page from app
- Register apply guidance page on root URL
- Remove overview page from acceptance tests
- Update acceptance tests to use apply guidance page as first page

### CODE REVIEW

File: src/test/common/page/guidance.js [hunk 1]
```diff
@@ -4,14 +4,23 @@ const Page = require('./page')
...
 class Guidance extends Page {
+  constructor (driver) {
+    super(driver)
+    this.sessionId = null
+  }
+
   async openGuidancePage (baseURL, pageName, lang = 'en') {
     try {
       await this.openPage(`${baseURL}${this.getPageWithTitle(PAGES, pageName).path}`, lang)
+      this.sessionId = await this.getCurrentSessionId()
```
Comment (gethyn1) [inline src/test/common/page/guidance.js]: Guidance pages to not use base page `open` method so session ID needs to be stored at this point

File: src/test/common/page/page.js [hunk 2]
```diff
@@ -32,7 +32,6 @@ class Page {
   async open (baseURL, lang = 'en') {
     try {
       await this.openDirect(baseURL, lang)
-      this.sessionId = await this.getCurrentSessionId()
```
Comment (mikeSmith08) [inline src/test/common/page/page.js]: You can also remove the line `this.sessionId = null` from the constructor in this file too.

File: src/test/smoke/features/smoke-test.feature [hunk 3]
```diff
@@ -3,7 +3,7 @@ Feature: Smoke test
   As the continuous delivery pipeline
   I want know that a new release has deployed correctly and is functional
 
-  Scenario: The overview page is available and starts the application process
-    Given I navigate to the Apply for Healthy Start overview page
+  Scenario: The guidance page is available and starts the application process
```
Comment (mikeSmith08) [inline src/test/smoke/features/smoke-test.feature]: I think this should start `The apply guidance page is available...`

File: src/test/common/page/page.js (hunk 2 above)
Comment (gethyn1) [reply src/test/common/page/page.js]: Updated

File: src/test/smoke/features/smoke-test.feature (hunk 3 above)
Comment (gethyn1) [reply src/test/smoke/features/smoke-test.feature]: Updated

**Pull request merged**